Conversions/BinaryToText.js

```diff
@@ -0,0 +1,12 @@
+// How to Convert Binary to Text
```

Comment:
This explanation is unnecessary, the code is readable as-is. If anything, add a proper JSDoc comment. Also, format this properly please.